### ISSUE TITLE
"Enable Steam Play for All Other Titles" not saving after Steam restart

### ISSUE THREAD

**Issue by fallout2077** [redmcg/wine#9]
I just installed this per the instructions on the site and I was wondering if it was normal behavior for the ckeckbox to clear following Steam restart when you change the "Run Other Titles With..." option.

**Comment by redmcg** [comment]
I get that too - but it still seems to apply. I think just because it's a custom version. Not sure if it's a bug or feature of Steam.

**Comment by fallout2077** [comment]
Okay then, thanks for the clarification! I'm starting to see several other issues rear their ugly heads today that weren't anywhere to be seen last night.

**Comment by redmcg** [comment]
Assuming the issues you're having are with other games, there's now an option in Steam to set a version for just the one game. You could try that?

![forceed](https://user-images.githubusercontent.com/8346438/53767995-a4a60c80-3f2b-11e9-86ea-53335b86f55d.png)


Edit: And then you can change your global setting back to the setting you were using previously.

**Comment by fallout2077** [comment]
Wow, I didn't know that! I'll give that a shot. The glaring problem at the moment is that the controls aren't working in-flight, and for peripherals such as an Xbox 360 controller, I can only move using the left analog stick. Plus, I am unable to enter the Controls menu... if I highlight and click on it, there's absolutely no indication of a keypress whatsoever. I tried to validate the game files and check for a Steam client update, though to no avail. I'll try what you mentioned above and see if there's any difference.

**Comment by redmcg** [comment]
Being unable to enter the Controls menu is a new issue that appears to have come up with a recent ED update. The only workaround known at the moment is to restart the client (some people say multiple times, but it usually works first shot for me). I have also seen someone say they made the game run in borderless window to fix it (I haven't tried that myself yet).

For the Xbox controller, you could try changing the 'Steam Input' setting (shown in screen-shot above) to 'Forced Off'.

**Comment by fallout2077** [comment]
Well, the controls menu worked again, though it may be a result of me having the Xbox controller disconnected at the time. I'll get back in and check. And I downloaded a custom .binds files off the net from ED Ref Card... aren't the binding files supposed to go into "ED_Pfx/drive_c/users/<user>/Local Settings/Application Data/Frontier Developments/Elite Dangerous/Options/Bindings/?

**Comment by redmcg** [comment]
That's correct. If you create a custom keybind - you'll find the it creates a file:
`pfx/drive_c/users/steamuser/Local Settings/Application Data/Frontier Developments/Elite Dangerous/Options/Binding/Custom.3.0.binds`

**Comment by fallout2077** [comment]
Hmm... I placed a couple keybind files from the net and from my Windows 10 install into that folder, but they weren't showing up in the game. Anyway, all that work was for nothing as my laptop has been shutting down while playing Elite... I switched to Linux as it seemed to be free of that, as I experienced it far more often in Win10, but while playing Elite within Linux, it shut down. So, I guess until I can get my laptop fixed or buy a new one, my only taste of Elite will be Reddit posts and YouTube videos by ObsidianAnt and Vindicator Jones. Thank you for your time helping me out, Mr. McGrath, and have a good day/night.
